### PR TITLE
SPUI-332 > Add navigation to buttons market details

### PR DESCRIPTION
**Pull request by VovaMedvediev**

### CODE REVIEW

File: lib/app/screens/market_details/provider/currency_pod.dart [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+import '../../../shared/models/currency_model.dart';
+import '../../../shared/providers/currencies_pod/currencies_pod.dart';
+
+final currencyPod = Provider.autoDispose.family<CurrencyModel, String>(
+  (ref, assetSymbol) {
+    final currencies = ref.read(currenciesPod);
+
+    return currencies.firstWhere((currency) => currency.symbol == assetSymbol);
```
Comment (eli1stark) [inline lib/app/screens/market_details/provider/currency_pod.dart:9]: can throw an error and end up being null just so you know. For these cases there is `orElse` close

**Comment by eli1stark** [comment]
Creating currencyPod is redundant.
Just listen for currencies in the BalanceActionButtons.
And create for example function currencyFrom(currencies, symbol) -> returns currency

**Review by eli1stark — CHANGES_REQUESTED** [review]
Creating currencyPod is redundant.
Just listen for currencies in the BalanceActionButtons.
And create for example function currencyFrom(currencies, symbol) -> returns currency

File: lib/app/screens/market_details/provider/currency_pod.dart (hunk 1 above)
Comment (VovaMedvediev) [reply lib/app/screens/market_details/provider/currency_pod.dart]: fixed

**Pull request merged**